### PR TITLE
[skip-ci] Packit: switch to EPEL instead of centos-stream+epel-next

### PR DESCRIPTION
**Pull request by lsm5**
- EPEL is the recommended target for further testing with rpm builds.
- Fix EL9 builds.
- Do not change c8s for now as it will be removed soon anyway.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by lsm5** [comment]
@containers/podman-maintainers PTAL.

The centos-stream+epel-next-9 builds have been failing on recent PRs because of `undefined: ecdsa.HashSign` which I suspect is an issue with the golang compiler that ended up in that environment. Anyway, epel-9* would be the right job environment to use if we want to add TMT tests and such, so this change would be required anyway.

**Comment by Luap99** [comment]
> The centos-stream+epel-next-9 builds have been failing on recent PRs because of undefined: ecdsa.HashSign which I suspect is an issue with the golang compiler that ended up in that environment. Anyway, epel-9* would be the right job environment to use if we want to add TMT tests and such, so this change would be required anyway.

yes see https://github.com/containers/skopeo/pull/2297#issuecomment-2059524991 and https://github.com/containers/skopeo/pull/2305

File: .packit.yaml [hunk 1]
```diff
@@ -47,8 +47,8 @@ jobs:
     targets:
       - centos-stream+epel-next-8-x86_64
       - centos-stream+epel-next-8-aarch64
-      - centos-stream+epel-next-9-x86_64
-      - centos-stream+epel-next-9-aarch64
+      - epel-9-x86_64
+      - epel-9-aarch64
```
Comment (Luap99) [inline .packit.yaml:51]: should the epel-next-8 above also changed?

**Comment by Luap99** [comment]
/lgtm
/hold

Feel free to remove the hold as you like.

File: .packit.yaml (hunk 1 above)
Comment (lsm5) [reply .packit.yaml:51]: We will be removing all el8 stuff once centos8 stream goes EOL end of next month, so let's leave it as is for now.

**Pull request merged**